### PR TITLE
feat: Add Source Sans Pro sans-serif font as default one

### LINKED ISSUE #8: Source Sans Pro font as default sans-serif font
[Source Sans Pro](https://fonts.google.com/specimen/Source+Sans+Pro) is default sans-serif Talend font from Talend styleguide.
I propose to embed [Source Sans Pro](github.com/adobe-fonts/source-sans-pro) as project Dependency.

### PR DESCRIPTION
**Pull request by frassinier**
Fix #8

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -1232,6 +1223,7 @@ <h4 class="modal-title">Source Code</h4>
 
     <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <script src="/bootstrap.js"></script>
```
Comment (acateland) [inline example/index.html]: shouldn't we get rid of those three js files ?

Comment (frassinier) [inline example/index.html]: Nop, indeed, this third import use temporarily useless! I will remove it!

**Comment by acateland** [comment]
lgtm

**Comment by talend-glorieux** [comment]
LGTM

**Pull request merged**